src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container-logo {
+  margin-left: 50px;
+}
+
+.logo {
+  display: flex;
+  padding: 10px 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: min-content;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  font-family: "Roboto", "Avenir", sans-serif;
+  font-size: 12px;
+  margin-right: 50px;
```

Comment:
your tests didn't pass because nav shouldn't have any outside margins, make a 50px horizontal padding for your header instead
```suggestion
```